docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
       DOCKER: "true"
     volumes:
       - .:/var/www/app
+      - /var/www/app/node_modules
```

Comment:
This ensures that the `node_modules` folder is ignored when volumes are mounted. Since the build already has it's own set of `node_modules` located at the same area, it will default to using that instead.